gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/model/PasswordPolicyEntity.java

```diff
@@ -19,12 +19,14 @@
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
+import java.util.List;
+
 @Getter
 @Setter
 @NoArgsConstructor
 public class PasswordPolicyEntity {
     private String id;
     private String name;
     private Boolean isDefault;
-    private Integer idpCount;
+    private List<String> idpsNames;
```

Comment:
**thought**; should we hav to provide a list of FilteredIdentityProvider to have at least the name, the id and the type of the IDP. In any cases, I this it should be better to have internalId instead of the name in a way to retrieve the right IDP as the name may not be unique

Reply:
ok